Makefile

```diff
@@ -1,5 +1,6 @@
 HYPERFINE := $(shell command -v hyperfine 2> /dev/null)
```

Comment:
not related with this PR, but this could be probably dropped

Reply:
Removed.